src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -105,17 +106,21 @@ void convertToCdf() {
       try {
         FileUtils.createOutputDirectory(config.getOutputDirectory());
         List<CastVoteRecord> castVoteRecords = parseCastVoteRecords(config, precinctIds);
-        ResultsWriter writer =
-            new ResultsWriter()
-                .setNumRounds(0)
-                .setContestConfig(config)
-                .setTimestampString(timestampString)
-                .setPrecinctIds(precinctIds);
-        try {
-          writer.generateCdfJson(castVoteRecords);
-        } catch (RoundSnapshotDataMissingException e) {
-          // This will never actually happen because no snapshots are involved when you're just
-          // translating the input to CDF, not the tabulation results.
+        if (castVoteRecords == null) {
```

Comment:
Is this addressing another open issue we have? If so, please include in the PR description.

Reply:
There's no issue, it improves the messaging on some bad inputs.